save-cloud-common/src/commonMain/kotlin/org/cqfn/save/domain/Role.kt

```diff
@@ -2,11 +2,11 @@ package org.cqfn.save.domain
 
 /**
  * User roles
- * @property string
+ * @property toPrint string representation of the [Role] that should be printed
  * @property priority
  */
 @Suppress("MAGIC_NUMBER", "MagicNumber")
-enum class Role(val string: String, val priority: Int) {
+enum class Role(val toPrint: String, val priority: Int) {
```

Comment:
Maybe `readableName` or `formattedName`?

Reply:
Yep, that is better. Too bad with naming, sorry